CMakeLists.txt

```diff
@@ -117,11 +117,16 @@ else (NOT GVM_STATE_DIR)
   set (GVM_STATE_DIR "${GVM_STATE_DIR}")
 endif (NOT GVM_STATE_DIR)
 
-if (NOT GVM_LOG_DIR)
-  set (GVM_LOG_DIR "${LOCALSTATEDIR}/log/gvm")
-else (NOT GVM_LOG_DIR)
-  set (GVM_LOG_DIR "${GVM_LOG_DIR}")
-endif (NOT GVM_LOG_DIR)
+if (NOT GSAD_LOG_FILE)
+  if (GVM_LOG_DIR)
+    set (GSAD_LOG_FILE    "${GVM_LOG_DIR}/gsad.log")
+  else (GVM_LOG_DIR)
+    set (GSAD_LOG_FILE    "-")
+  endif (GVM_LOG_DIR)
+else (NOT GSAD_LOG_FILE)
+  set (GSAD_LOG_FILE    "${GSAD_LOG_FILE}")
```

Comment:
Is this case really needed, setting to itself? I see GVM_STATE_DIR is doing it but if it's required then surely DATADIR would also be doing it.